### DISCUSSION TITLE
Usage of nested tags (children)

### DISCUSSION THREAD

**Discussion by YoSev** [markdoc/markdoc#296] [General]
Hello,

i am using markdoc with vue3 and stumbled up on a question how it would be possible to use nested tags to mimic the style of ```{% table %}```.

Our current apporach is working but its very hard to type:
```
{% timeline %}

{% timelineHeader %}
2022/04
{% /timelineHeader %}
{% timelineItem type="BUG" %}
changed a
{% /timelineItem %}
{% timelineItem type="FEATURE" %}
changed b
{% /timelineItem %}
{% timelineItem type="CHANGE" %}
changed c
{% /timelineItem %}

{% timelineHeader %}
2022/03
{% /timelineHeader %}
{% timelineItem type="BUG" %}
changed a
{% /timelineItem %}
{% timelineItem type="FEATURE" %}
changed b
{% /timelineItem %}
{% timelineItem type="CHANGE" %}
changed c
{% /timelineItem %}

{% timelineHeader %}
2022/02
{% /timelineHeader %}
{% timelineItem type="BUG" %}
changed a
{% /timelineItem %}
{% timelineItem type="FEATURE" %}
changed b
{% /timelineItem %}
{% timelineItem type="CHANGE" %}
changed c
{% /timelineItem %}

{% /timeline %}
```  
it results into:
![grafik](https://user-images.githubusercontent.com/1394374/203604041-78fa64f6-44f2-4629-85d4-7c9eadd2e7f0.png)

i would prefer a style like this:
```
{% timeline %}
* Heading 1
- Row 1 Cell 1 {% type="CHANGE" %}
- Row 1 Cell 2
* Heading 2
- Row 2 Cell 1
- Row 2 cell 2
{% /timeline %}
```

Does this require to create a custom renderer?

**Comment by mfix-stripe** [comment]
This just requires a custom [`timeline` tag transform function ](https://markdoc.dev/docs/tags) — no customer renderer needed. 🙂

**Reply by YoSev** [reply]
Thanks.

This will override the actual render component, right?
So it is one or the other, do i get this right from the documentation?
